do-dont/stories/index.stories.js

```diff
@@ -108,3 +108,35 @@ export const do_custom_color = () => `
     </ul>
   </dockit-do>
 `;
+
+export const do_themed = () =>
+  html`
+    <style>
+      ${css`
+        .themed-do dockit-do::part(container) {
+          border-style: solid;
+          border-width: 2px;
+          border-color: #000080;
+          border-radius: 1rem;
+        }
+        .themed-do dockit-do::part(component-container) {
+          background-color: #00008020;
+        }
+        .themed-do dockit-do::part(instructions-container) {
+          background-color: #00008040;
+        }
+      `}
+    </style>
+    <div class="themed-do">
+      <dockit-do color="#000080" label="NAVY BLUE">
```

Comment:
the more I look at `dockit-do` examples, the more I feel like it's bad to have `color` attribute
it should be in CSS

most likely users will prefer to style it once for the entire doc site in the following manner:

```css
dockit-do {
  --dockit-do-color: #123123;
}
```

attribute API is much less handy, not sure if there is a good use case for it

Reply:
Yes the `do` and `don't` colors should be managed centrally via CSS only.

Reply:
Like what you did with parts for the rest  👍